lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -907,7 +912,8 @@ def _host_config(self):
             devices='devices',
             pid_mode='pid_mode',
             tmpfs='tmpfs',
-            init='init'
+            init='init',
+            runtime='runtime'
```

Comment:
Similar to #40839, you should make sure to only insert this when needed, so older versions of the `docker`/`docker-py` modules which don't support this won't choke on it. (It doesn't seem to have been supported by docker-py 1.7.0.)
Probably it's best to wait until #40839 is merged, and then copy its approach, as otherwise you'll get merge conflicts as the line above will be modified by it.

Reply:
#40839 has been merged.